kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -356,11 +349,19 @@ class SolvingStrategy
 
     /**
      * @brief Operations to get the pointer to the model
-     * @return mpModelPart: The model part member variable
+     * @return *mpModelPart: The model part member variable
      */
-    inline ModelPart& GetModelPart()
+    ModelPart& GetModelPart()
+    {
+        return *mpModelPart;
```

Comment:
also it should be checked here that the pointer is valid, otherwise you get a segfault!

Reply:
Yes. Indeed the current implementation does it. I can't understand why I did it in this way. Thanks!

Reply:
Done.